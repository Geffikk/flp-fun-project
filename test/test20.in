S,Z
a
S
S->Sa
S->Z
Z->S
