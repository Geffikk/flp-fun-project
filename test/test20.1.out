S,Z
a
S
S->Sa
Z->Sa
